android/src/main/java/com/example/flutternativeimage/FlutterNativeImagePlugin.java

```diff
@@ -105,10 +105,17 @@ public void onMethodCall(MethodCall call, Result result) {
       BitmapFactory.Options options = new BitmapFactory.Options();
       options.inJustDecodeBounds = true;
       BitmapFactory.decodeFile(fileName,options);
-
       HashMap<String, Integer> properties = new HashMap<String, Integer>();
-      properties.put("width",options.outWidth);
-      properties.put("height",options.outHeight);
+      properties.put("width", options.outWidth);
+      properties.put("height", options.outHeight);
+
+      try {
+        ExifInterface exif = new ExifInterface(fileName);
```

Comment:
nitpick: I don't like that everything is inside this try/catch block. It would be good that only code that can fail will be handled. I don't know what could fail, but I'd do something like this:

```java
int orientation = 0;
properties.put("orientation", orientation);

ExifInterface exif = null;

try {
  // assuming this can throw an exception
  exif = new ExifInterface(fileName);  
} catch (IOException ex)
  // why can this happen?
}

if(exif != null) {
  orientation = exif.getAttributeInt(ExifInterface.TAG_ORIENTATION, ExifInterface.ORIENTATION_UNDEFINED);
  properties.put("orientation", orientation);
}
```